### PR TITLE
Fix class confirm message

### PR DESCRIPTION
**Pull request by KingOfSquares**
Fixes this bug:

Didn't test but I mean :shrug:
Signed-off-by: KingSimon <19822231+KingOfSquares@users.noreply.github.com>

### CODE REVIEW

File: core/src/main/java/tc/oc/pgm/picker/PickerMatchModule.java [hunk 1]
```diff
@@ -694,8 +694,9 @@ private void handleInventoryClick(
             cmm.setPlayerClass(player.getId(), cls);
             player.sendMessage(
                 text()
-                    .append(translatable("match.class.ok", NamedTextColor.GOLD))
-                    .append(text(name, NamedTextColor.GREEN))
+                    .append(
+                        translatable("match.class.ok", NamedTextColor.GOLD),
+                        text(name, NamedTextColor.GREEN))
```
Comment (Pablete1234) [inline core/src/main/java/tc/oc/pgm/picker/PickerMatchModule.java]:
```suggestion
                        translatable("match.class.ok", NamedTextColor.GOLD,
                        text(name, NamedTextColor.GREEN))
```

**Review by Pablete1234 — CHANGES_REQUESTED** [review]
You should probably test it

File: core/src/main/java/tc/oc/pgm/picker/PickerMatchModule.java (hunk 1 above)
Comment (KingOfSquares) [reply core/src/main/java/tc/oc/pgm/picker/PickerMatchModule.java]: wait... 🤔

**Comment by KingOfSquares** [comment]
Lets pretend like I managed to do this first try....

**Review by Pablete1234 — APPROVED** [review]

**Review by Electroid — APPROVED** [review]

**Pull request merged**